_drafts/2017-06-07-checkers.md

```diff
@@ -0,0 +1,115 @@
+---
+name: checkers
+layout: post_discourse
+title: Tackling the Research Compendium at #runconf17
```

Comment:
@noamross @sckott I'd like to have this posted on main blog 2017-06-20 (could go live Monday night with that date I think) and I will tweet it from \@ropensci

Thank you to team `checkers` for this!